engine/src/main/java/org/camunda/bpm/engine/runtime/JobQuery.java

```diff
@@ -32,12 +33,18 @@
   /** Only select jobs with the given id */
   JobQuery jobId(String jobId);
 
+  /** Only select jobs whose id is in the given set of ids */
+  JobQuery jobIds(Set<String> ids);
```

Comment:
Let's cover this public API by a dedicated test case. Please also do for line 46.

Reply:
Tests added 👍